src/components/panels/request/RequestBody.tsx

```diff
@@ -61,6 +68,18 @@ export function RequestBody({ request }: RequestBodyProps) {
 								} else {
```

Comment:
This is something we can do later, but I'd like to pull all this out into its own function since the logic is so complex now.